frontend/public/style/_icons.scss

```diff
@@ -26,6 +27,7 @@
   color: $color-red-error;
 }
 
-.alert-pending {
+.alert-pending,
+.silence-pending {
```

Comment:
Should this be the warning orange or text-muted gray? (I don't know alert manager well enough to say.)

Reply:
The "expired" Silence icon is gray, so I think it makes sense to leave it as orange to more easily see the difference.